### ISSUE TITLE
importPath ".." causing excessive build times

### ISSUE THREAD

**Issue by lionello** [CyberShadow/ae#52]
https://github.com/CyberShadow/ae/blob/3a2ec9456c1a6568b2ef9fa909c5f29cd1b0d829/dub.sdl#L13

This causes issues when building an application that depends on `ae` from Nix: the `ae` library will have been added to the Nix store (`/nix/store/…`) but because of the `importPath ".."`, `dub` will iterate all files and folders in the Nix store. This is often 20GiB worth of packages, and doesn't complete (I aborted after 5 minutes.)

**Comment by CyberShadow** [comment]
Yes, this is due to a problem in the Dub package manager. See commit 363227fb3953276f0d6e1c0eb2ef7d2e0118a5e6 for details. It also describes this exact problem and a workaround.

**Comment by CyberShadow** [comment]
Discussion on a fix in Dub: https://github.com/dlang/dub/pull/2637
